src/Car.java

```diff
@@ -1,34 +1,16 @@
-// Класс для автомобилей
-public class Car implements Vehicle {
-    private String modelName;
-    private int wheelsCount;
-
-    @Override
-    public String getModelName() {
-        return modelName;
+public class Car extends Transport implements Vehicle {
+    public Car(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
     }
 
     @Override
-    public void setModelName(String modelName) {
-        this.modelName = modelName;
-    }
-
-    @Override
-    public int getWheelsCount() {
-        return wheelsCount;
-    }
-
-    @Override
-    public void setWheelsCount(int wheelsCount) {
-        this.wheelsCount = wheelsCount;
-    }
-
-    @Override
-    public void updateTyre() {
-        System.out.println("Меняем покрышку");
-    }
-
-    public void checkEngine() {
-        System.out.println("Проверяем двигатель");
+    public void service() {
+        System.out.println("Обслуживаем автомобиль " + getModelName());
+        int wheelsCount = getWheelsCount();
+        for (int i = 0; i < wheelsCount; i++) {
+            System.out.println("Меняем покрышку " + (i+1));
+        }
+        System.out.println("Проверка двигателя");
```

Comment:
```suggestion
        System.out.println("Проверка двигателя машины");
```
чтобы подчеркнуть, что это не так, как для грузовика, даже если код один и тот же